### PR TITLE
Actions and Version Requirements

### PR DESCRIPTION
**Pull request by limenet**
While using this package in a Pimcore context, I encountered a few rough edges, some of which have already been fixed, e.g. #42 -- thanks! Currently, the automated tests do not work since Travis CI no longer offers that service and the Node.js requirements are stricter than what is required by Puppeteer itself. Additionally, there were some font rendering issues which are fixed by adding a CLI flag. This PR addresses those aspects. I hope that's okay with you :)

Thanks for your package and let me know if you'd like me to make any changes!

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: PHP Checks
+
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
+
+jobs:
+
+  phpstan_pimcore10:
```
Comment (spiritix) [inline .github/workflows/php.yml:10]: Please rename this to something neutral

File: lib/Converter.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ class Converter
     {
         return puppeteer.launch({
             ignoreHTTPSErrors: true,
-            args: ['--no-sandbox', '--disable-web-security']
+            args: ['--no-sandbox', '--disable-web-security', '--font-render-hinting=none']
```
Comment (spiritix) [inline lib/Converter.js:48]: What issue is this intended to fix?

File: phpunit.xml [hunk 3]
```diff
@@ -7,19 +7,18 @@
...
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist>
-            <directory>./src</directory>
-        </whitelist>
-    </filter>
-</phpunit>
\ No newline at end of file
+         stopOnFailure="false">
```
Comment (spiritix) [inline phpunit.xml:10]: Please undo, the indents are now double

**Review by spiritix — CHANGES_REQUESTED** [review]

File: lib/Converter.js (hunk 2 above)
Comment (limenet) [reply lib/Converter.js:48]: Unfortunately I can't share any screenshots, but it fixes this issue: https://github.com/puppeteer/puppeteer/issues/2410

File: .github/workflows/php.yml (hunk 1 above)
Comment (limenet) [reply .github/workflows/php.yml:10]: Absolutely, copy-pasted this from another project

File: phpunit.xml (hunk 3 above)
Comment (limenet) [reply phpunit.xml]: Not sure if I understood you correctly, but should be fixed now

**Review by spiritix — APPROVED** [review]

**Pull request merged**